docs/dashboard/dashboard.rst

```diff
@@ -0,0 +1,275 @@
+For example, the resource in the image below will deploy a file with path `/etc/my.cnf` and file permission `644`.
+We can even inspect the file's content.
+
+.. figure:: /dashboard/images/img_10.png
+   :width: 100%
+   :align: center
+   :alt: Resource view
+
+   Resource desired state view
+
+Below is the action log, shows a log of actions taken on the given resource.
+This varies from dry-runs to deploys.
+This log will typically start filling up with deploys due to the orchestrator enforcing the desired state.
+Again we can further inspect an action by pressing the drop down arrow.
+
+.. figure:: /dashboard/images/img_11.png
+   :width: 100%
+   :align: center
+   :alt: Resource action
+
+   Resource action view
+
+Each of these logs can then be further analyzed by pressing the ``magnifying glass``.
+
+The Resources Overview
+----------------------
+
+The resources overview, not to be confused with the similar resource version overview, gives an overview of all known resources.
+This is not only for resources of the currently deployed model, but potentially resources from older models and the state they are in.
+
+.. figure:: /dashboard/images/img_12.png
+   :width: 100%
+   :align: center
+   :alt: Resources Overview
+
+   The Resources Overview
+
+While not as in depth as the resource version overview, it does link every resource to its deployed version, so the resource can be inspected there.
+
+The Parameters View
+-------------------
+
+The parameter overview gives a list of parameters.
+Parameters are part of the model, but their value may or may not be known at compile time.
+For example, the IP address of a virtual machine that is created by the model.
+
+.. figure:: /dashboard/images/img_13.png
+   :width: 100%
+   :align: center
+   :alt: Parameter overview
+
+   The Parameter overview
+
+Each parameter can be individually inspected or deleted.
+Inspecting the resource allows us to read additional metadata if any is available.
+
+The Agent Overview
+------------------
+
+The agent overview shows different agents and the state they are in.
+
+.. figure:: /dashboard/images/img_14.png
+   :width: 100%
+   :align: center
+   :alt: Agent overview
+
+   The Agent overview
+
+This overview allows us to ``Force deploy`` and ``Force repair`` resources on a per agent basis.
+Pausing an agent stops deployments for that agent.
+Useful when, for example, diagnosing problems on the machine the agent deploys to, without having to stop enforcement of the whole model.
+
+The Agent Processes overview, lists the different processes running agents.
+The magnifying glass allows us to inspect each process in more detail:
+
+.. figure:: /dashboard/images/img_15.png
+   :width: 100%
+   :align: center
+   :alt: Agent process inspection
+
+   Agent process inspection
+
+Here we can find the process ``pid``, the ip addresses the server has bound to and what version of python inmanta is running on, amongst other things.
```

Comment:
```suggestion
Here we can find the process ``pid``, the ip addresses the server has bound to and what version of Python inmanta is running on, among other things.
```

Reply:
amongst isn't incorrect here. They are interchangeable.

Reply:
Indeed, didn't know that, and my lexical corrector didn't either.

Reply:
Fixed